.travis.yml

```diff
@@ -1,8 +1,8 @@
+dist: xenial
+services:
+  - xvfb
 language: node_js
 node_js:
-  - stable
+  - 10
```

Comment:
Going to node 12 breaks gulp, this is the highest we can use for the time being.